openshift/template.yaml

```diff
@@ -40,6 +40,10 @@ objects:
                   value: ${KEEP_WORKER_RESULT_NUM_DAYS}
                 - name: GENERATE_MANIFESTS
                   value: ${GENERATE_MANIFESTS}
+                - name: SENTRY_API_ISSUES
+                  value: ${SENTRY_API_ISSUES}
+                - name: SENTRY_API_TAGS
+                  value: ${SENTRY_API_TAGS}
```

Comment:
provide the default macro value in the template as well
example
- description: Dry run
  displayName: Dry run
  required: true
  name: DRY_RUN
  value: "false"

Reply:
Done, 
Thanks @yzainee